src/widgets/Menu/Menu.tsx

```diff
@@ -66,11 +66,14 @@ const Menu: React.FC<NavProps> = ({
   toggleTheme,
   langs,
   setLang,
+  toggleNotPossible,
```

Comment:
I don't follow what `toggleNotPossible` means.

Reply:
Toggle not possible is for saying don't have toggle (light/dark).    In DEX we don't allow light mode currently.